.github/tests/production-ingress-nginx/post-install.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -x
+
+SCRIPT=$(readlink -f "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+
+k_wait=(kubectl wait --for condition=available --timeout 30s --namespace)
+k_rollout_status=(kubectl rollout status --watch --timeout 30s --namespace)
```

Comment:
Maybe do the same fix as in your recent PR. To align the style now we are touching it newly